### PR TITLE
Adds the intern

### PR DESCRIPTION
**Pull request by optimumtact**
The intern is just the name of a user who is playing the assistant job
but has less than ten living hours

## Why It's Good For The Game

This is to help you identify who is a new player, and who is a burn out
player who just wants to grief you. This way you know who needs help
and who can be safely ignored or turned into a nugget

## Changelog
:cl: oranges
add: Assistants with less than 10 hours are now called Interns on their job ID
/:cl:

### CODE REVIEW

**Comment by Iatots** [comment]
Why not for every basic job?

**Comment by optimumtact** [comment]
I don't feel like writing the code to check each job's specific EXP type, feel free to do it

**Comment by ATH1909** [comment]
just add (Intern) in parenthesis after your job title if you have less than 10 hours in the game

**Review by Rohesie — CHANGES_REQUESTED** [review]
Can you modify `sec_hud_set_ID()` so this new job gets the assistant hud icon, or else introduce a new intern icon?

**Comment by Jack7D1** [comment]
Would you mind upping the time to 15h? I was a pretty slow learner myself.

**Comment by PKPenguin321** [comment]
Putting an intern tag on any job for a new player is a better idea since plenty of new players will roll for jobs by accident/out of ignorance

File: code/modules/jobs/job_types/assistant.dm [hunk 1]
```diff
@@ -17,13 +17,22 @@ Assistant
...
 	display_order = JOB_DISPLAY_ORDER_ASSISTANT
+	var/new_player_title = "Intern"
 
 /datum/job/assistant/get_access()
 	if(CONFIG_GET(flag/assistants_have_maint_access) || !CONFIG_GET(flag/jobs_have_minimal_access)) //Config has assistant maint access set
 		. = ..()
 		. |= list(ACCESS_MAINT_TUNNELS)
 	else
 		return ..()
+/datum/job/assistant/get_title(mob/H)
```
Comment (Jared-Fogle) [inline code/modules/jobs/job_types/assistant.dm:28]: Should have leading new line?

File: code/modules/jobs/job_types/_job.dm [hunk 2]
```diff
@@ -169,6 +169,9 @@
 
 	return max(0, minimal_player_age - C.player_age)
 
+/datum/job/proc/get_title(var/mob/living/H)
```
Comment (Jared-Fogle) [inline code/modules/jobs/job_types/_job.dm:172]:
```suggestion
/datum/job/proc/get_title(mob/living/H)
```

Comment (tralezab) [reply code/modules/jobs/job_types/_job.dm:172]: it's really up to preference

Comment (Jared-Fogle) [reply code/modules/jobs/job_types/_job.dm:172]: Was pretty sure this was in the style guide, oh well.

**Comment by Tlaltecuhtli** [comment]
intern should be for <500h newfriends

**Comment by anconfuzedrock** [comment]
I think 10 living hours could do with being doubled because this game has a steep learning curve but on the other hand ehhhhhhhhhhhh

**Comment by Cyberboss** [comment]
Exquisite PR.

File: code/modules/jobs/job_types/assistant.dm [hunk 3]
```diff
@@ -17,13 +17,22 @@ Assistant
...
+	var/final = title
+	var/client/the_client = H.client
```
Comment (AnturK) [inline code/modules/jobs/job_types/assistant.dm:30]: Client will only be here for late joiners, you'd need to pass down client from job equip down for roundstart ones

**Comment by optimumtact** [comment]
not interested in the bikeshedders, go make your own PR's

File: code/modules/jobs/job_types/assistant.dm (hunk 3 above)
Comment (optimumtact) [reply code/modules/jobs/job_types/assistant.dm:30]: god that's annoying

**Comment by PKPenguin321** [comment]
I prefer the term "idea guy" when it comes to codeslurs thank you very much

File: code/modules/jobs/job_types/_job.dm (hunk 2 above)
Comment (Rohesie) [reply code/modules/jobs/job_types/_job.dm:172]: Not having `var` in it is preferable, yes.

**Comment by Cyberboss** [comment]
Just to be sure, does this properly affect the job select screen?

**Comment by optimumtact** [comment]
no it doesn't, it appears as assistant, please read the code before asking

**Comment by TheVekter** [comment]
Y'all know one of us is going to eventually remove assistant because of this, right?

**Comment by Iatots** [comment]
You could just as easily lock the role for anyone over 50 hours.
It's been tried before, department tide is worse than greytide.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Antur + Jared

**Comment by Naloacisamazing** [comment]
change it to 20 hours. Alot of new players stop and start and could still need help for a good while after 10 hours

**Comment by optimumtact** [comment]
@TheVekter there will always be a job for people who don't want to do anything

**Comment by Coastfront** [comment]
up the amount of hours to 200 living so that we can have 
Small Cock (captain) (intern)

**Pull request closed without merging**